security/subsystem/src/main/java/org/jboss/as/security/AuditResourceDefinition.java

```diff
@@ -54,7 +54,7 @@ public void registerAttributes(final ManagementResourceRegistration resourceRegi
     @Override
     public void registerChildren(ManagementResourceRegistration resourceRegistration) {
         super.registerChildren(resourceRegistration);
-        resourceRegistration.registerSubModel(new MappingProviderModuleDefinition(Constants.PROVIDER_MODULE));
```

Comment:
What happened to MappingProviderResourceDefinition?  From what I see this is the only place it is used but unless github is not showing correctly it has not been removed.